src/sentry/static/sentry/app/views/organizationGroupDetails/actions.jsx

```diff
@@ -133,6 +135,18 @@ const GroupDetailsActions = createReactClass({
     return `${protocol}//${host}${path}`;
   },
 
+  getDiscoverUrl() {
```

Comment:
@leedongwei I prefer these Discover2 URLs to be generated thru the EventView class. Specifically: https://github.com/getsentry/sentry/blob/15d06803b5e44fc49b53b2b60407e0dabea6c298/src/sentry/static/sentry/app/views/eventsV2/eventView.tsx#L382

then generate the query object in https://github.com/getsentry/sentry/blob/15d06803b5e44fc49b53b2b60407e0dabea6c298/src/sentry/static/sentry/app/views/eventsV2/eventView.tsx#L526

We'll want to do this in case the Discover2 schema changes again in the future (I wouldn't be surprised if it does soon). For example, recent changes to the schema now include the removal of `fieldnames` and `tags`.

-----

We used to have another dedicated static method to generate EventView objects for the pre-built queries, but it has since been merged to `static fromSavedQuery()`.

Reply:
Fair point.